amethyst_ui/src/prefab.rs

```diff
@@ -68,6 +68,7 @@ pub struct UiTransformBuilder<G> {
     /// UI element with the next highest tab order, or if another element with the same tab_order
     /// as this one exists they are ordered according to Entity creation order.  Shift-tab walks
     /// this ordering backwards.
+    pub tab_order: u32,
```

Comment:
I'm a bit confused by this field. Why was this missing before? And, do we even use it anywhere?

Anyway, the build failed because `selectable` below does not have any API docs.

Reply:
This is a change that I didn't mean to commit, my apologies for the extra work.  

The UI prefabs found in `examples/assets/ui` have tab_order fields all over the place.

I was originally going to add the missing fields, as in the change above, but decided to revert that change and keep the scope of this commit to the RON files.

Reply:
IIRC tab_order was renamed to selectable or something similar.